ClickHouseClient/ClickHouseClient.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
```

Comment:
What is the purpose of the ClickHouseClient and all the rest of the files in this folder? Is this some sort of test?